Docs: fix out-dated CLI option in Manim's Output Settings (#3674)
ManimCommunity/manim: docs/source/tutorials/output_and_config.rst

   manim -pql scene.py SquareToCircle

> it specifies the scene to render.  This is not necessary now.  When a single file contains only one ``Scene`` class, it just render the only one.  When a single file contains more than one ``Scene`` class, you can leave a blank, and the manim will let you choose a ``Scene`` class.  If your file contains multiple ``Scene`` classes, and you want to render them all, you can use the ``-a`` flag.

Reviewer: ```suggestion
it specifies the scene to render.  This is not necessary now.  When a single file contains only one ``Scene`` class, it will render that class.  When a single file contains more than one ``Scene`` class, manim will let you choose a ``Scene`` class.  If your file contains multiple ``Scene`` classes, and you want to render them all, you can use the ``-a`` flag.
```

   manim -pql scene.py SquareToCircle

it specifies the scene to render.  This is not necessary now.  When a single
file contains only one ``Scene`` class, it will just render the ``Scene``
class.  When a single file contains more than one ``Scene`` class, manim will

> let you choose a ``Scene`` class.  If your file contains multiple ``Scene``

Reviewer: ```suggestion
let you choose a ``Scene`` class. If your file contains multiple ``Scene``
```
Sorry missed this change earlier!

   manim -pql scene.py SquareToCircle

it specifies the scene to render.  This is not necessary now.  When a single

> file contains only one ``Scene`` class, it just render the only one. When a

Reviewer: ```suggestion
file contains only one ``Scene`` class, it will just render the ``Scene`` class. When a
```
Seems your force push broke your previous commit?

Author: There was a typo in previous commit message, so I push -f to fix it. Sorry for that😪
